sdk/package.json

```diff
@@ -81,8 +81,9 @@
   "scripts": {
     "clean": "rm -rf ./dist",
     "copy-files": "cp -r ./proto ./dist && cp index.d.ts ./dist",
-    "compile": "tsc && npm run copy-files",
-    "prepack": "npm run compile",
+    "compile": "tsc --declaration false --composite false && npm run copy-files",
```

Comment:
Since we don't need two versions of the `tsc` command anymore, we can move the `declaration` and `composite` flags to the `tsconfig.json` config file.

Reply:
Good idea. Done.